networkit/cpp/generators/MocnikGenerator.h

```diff
@@ -19,7 +19,12 @@ namespace NetworKit {
  */
 class MocnikGenerator: public StaticGraphGenerator {
 private:
+	// The node position map contains, for each node, a pair consisting of the node
+	// itself in the first component and its position in space in the second
+	// component.
 	typedef std::vector<std::pair<node, std::vector<double>>> NodePositionMap;
```

Comment:
Thank you for the comment. Is the first component not already deducible from the index?

Reply:
I am not sure about the type `node`.  If it is clear that the type `node` is (and will ever) be of the same type as the index of the vector, then yes.

Reply:
Yes. `node` is an index type between zero and `G.upperNodeIdBound()` (which can differ from `G.numberOfNodes()` if nodes are deleted).